README.md

```diff
@@ -841,6 +841,61 @@ Going crazy:
 ; => {:x 24}
 ```
 
+## To and from JSON
+
+The `m/encode` and `m/decode` functions work on clojure data. To go
+from clojure data to JSON, you need a JSON library like
+[jsonista](https://github.com/metosin/jsonista). Additionally, since
+`m/decode` doesn't check the schema, you need to run `m/validate` (or
+`m/explain`) if you want to make sure your data conforms to your
+schema.
+
+To JSON:
+
+```clj
+(def Tags
+  [:map
```

Comment:
this could be wrapped into `m/schema` (as a good practice) to enable caching of validators & explainers.

Reply:
done!